null.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+var RingBuffer = require('ringbufferjs');
+
+module.exports = NullStatsd;
+
+function NullStatsd(capacity) {
```

Comment:
I think that's out of scope. 

I also don't agree with testing using spies. Testing by asserting against `this._buffer` like I'm doing in my tests is a cleaner solution.